src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/MsQuicNativeMethods.cs

```diff
@@ -1081,9 +1039,9 @@ internal uint ListenerOpen(SafeMsQuicRegistrationHandle registration, ListenerCa
 
                 return __retVal;
             }
-            internal uint ListenerStart(SafeMsQuicListenerHandle listener, QuicBuffer* alpnBuffers, uint alpnBufferCount, ref SOCKADDR_INET localAddress)
+            internal uint ListenerStart(SafeMsQuicListenerHandle listener, QuicBuffer* alpnBuffers, uint alpnBufferCount, byte* localAddress)
             {
-                IntPtr __listener_gen_native = default;
+                IntPtr __listener_gen_native;
```

Comment:
Compiler complained that the default-assigned value was never used

Reply:
We still have it like that in other places since this is source generated code, so could we leave it as it was? Unless it breaks the build due to `warnaserror` or something like that.

Reply:
it breaks the build otherwise